.github/workflows/linters.yml

```diff
@@ -0,0 +1,62 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
+    name: Webhint
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Webhint
+        run: |
+          npm install --save-dev hint@6.x
+          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.hintrc
+      - name: Webhint Report
+        run: npx hint --telemetry=off .
```

Comment:
Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.

Reply:
I didn't.
The file is exactly the same as the one on the microverse linters repo.
I cloned the repo and copied the file.

I annexed a print from the micrverse repo and you tell me what is different
![Screenshot_2021-04-29 microverseinc linters-config](https://user-images.githubusercontent.com/69148821/116634926-d06a8480-a933-11eb-84ce-67e161bf1f75.png)

Reply:
So Explain to me what is wrong again? Why did you waste my time invalidating a request claiming that I changed the linters configuration?